src/components/Uploader/assets/cross-device-small-icon.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Has this been optimised?

Reply:
Yeap. :)

BTW I think we should consider having image optimisation as part of the Webpack build using `image-webpack-loader` as this is easy to forget/miss both during development and review.

Reply:
I agree